src/Libraries/RevitNodes/Elements/CurtainPanel.cs

```diff
@@ -154,7 +158,9 @@ public bool HasPlane
             return true;
          }
       }
-
+       /// <summary>
+       /// Gets panel plane from the given curtain panel
```

Comment:
In this method, there are a lot of strings used, can you please move them to resource file.